src/react/account/details/Properties.jsx

```diff
@@ -2,50 +2,80 @@
 
 import Table, * as T from '../../ui-elements/TableKeyValue';
 import type { Account } from '../../../types/account';
```

Comment:
These two pages seem to have the same layout. Do you mind reusing TableContainer and ButtonContainer components? https://github.com/scality/zenko-ui/blob/development/1.0/src/react/databrowser/buckets/details/Overview.jsx

Reply:
done